src/container/srv_target.c

```diff
@@ -1812,7 +1812,7 @@ cont_snap_update_one(void *vin)
 		size_t	 bufsize;
 
 		bufsize = args->snap_count * sizeof(*args->snapshots);
-		D_REALLOC(buf, cont->sc_snapshots, bufsize);
+		D_REALLOC_NZ(buf, cont->sc_snapshots, bufsize);
```

Comment:
Just a comment on this, but it looks like you could use cont->sc_snapshots directly here, and REALLOC_ARRAY rather than REALLOC

Reply:
Indeed.   Will look at a followup patch